apps/openmw/mwrender/renderingmanager.cpp

```diff
@@ -347,6 +341,13 @@ namespace MWRender
         // It is unnecessary to stop/start the viewer as no frames are being rendered yet.
         mResourceSystem->getSceneManager()->getShaderManager().setGlobalDefines(globalDefines);
 
+        bool forceShaders = Settings::Manager::getBool("radial fog", "Shaders")
+                            || Settings::Manager::getBool("force shaders", "Shaders")
```

Comment:
As this is being moved anyway, it's probably a good idea to put `force shaders` at the top.